src/Catalyst.Core.Modules.Web3/Controllers/EthController.cs

```diff
@@ -47,9 +49,33 @@ public EthController(IWeb3EthApi web3EthApi, IWeb3HandlerResolver handlerResolve
             _handlerResolver = handlerResolver ?? throw new ArgumentNullException(nameof(handlerResolver));
             _jsonSerializer = jsonSerializer ?? throw new ArgumentNullException(nameof(jsonSerializer));
         }
+        
+        private JsonRpcResponse[] HandleManyRequests(JsonRpcRequest[] requests)
+        {
+            JsonRpcResponse[] responses = new JsonRpcResponse[requests.Length];
+            for (int i = 0; i < requests.Length; i++)
+            {
+                responses[i] = HandleSingleRequest(requests[i]);
+            }
+
+            return responses;
+        }
 
         [HttpPost]
-        public JsonRpcResponse Request([FromBody] JsonRpcRequest request)
+        public object Request([FromBody] object body)
+        {
+            JObject jObject = body as JObject;
```

Comment:
jObject returns null if body is an array. Should it be JToken?

Reply:
fixed, thanks, we need to add tests for this